snmp/README.md

```diff
@@ -165,6 +165,7 @@ In Linux, use this format for the script:
 
```

Comment:
Seems there is one more case here: https://github.com/DataDog/integrations-core/pull/4159/files#diff-04629e6b73db9025cd3d09b57a18b715R153

Reply:
Yes, taht is the one I mention on the description, but that is the output of `dir mibdump.py /s`, right?

Reply:
Yes, right, seems so, no change then.